ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -325,6 +389,10 @@ export default function Page() {
             columns={columns}
             items={data?.tasks_status || []}
```

Comment:
I think we should use `newRows` as items, because the the `startIndex` filed of `groups` depends on the data in the `newRows` instead of the `data?.tasks_status`.

We can have an experiment, use `['tifalsh', 'tidb', 'tikv', 'pd']` replace `['pd', 'tidb', 'tikv', 'tiflash']` in the line 244, then we will get wrong result:

![image](https://user-images.githubusercontent.com/1284531/146499992-9db6258c-b13e-4dd0-a3c6-d69c420aaf41.png)

`2379` is the PD component port.